pkg/releaseutil/manifest.go

```diff
@@ -42,8 +42,13 @@ func SplitManifests(bigfile string) map[string]string {
 	res := map[string]string{}
 	// Making sure that any extra whitespace in YAML stream doesn't interfere in splitting documents correctly.
 	docs := sep.Split(bigfile, -1)
-	for i, d := range docs {
-		res[fmt.Sprintf(tpl, i)] = d
+	var count int = 0
+	for _, d := range docs {
```

Comment:
added TrimSpace in two spots just in case we run into a situation where there is extra white space. This was taken out before and I don't know why so that's the only thing that makes me nervous. Might have been an accident though.